contracts/MerkleFactory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "./MerkleDrop.sol";
+
+contract MerkleFactory {
+    MerkleDrop[] public drops;
+    event CreateDrop(address dropAddress);
+
+    function createDrop(
```

Comment:
Can anyone execute `createDrop` function or only the proposal creators?

Reply:
Anyone can access it. Validating if the caller is a proposal creator is tricky and it keeps on increasing. Do you have any suggestions to tackle this?